test/e2e/network/service.go

```diff
@@ -1552,7 +1552,6 @@ var _ = SIGDescribe("Services", func() {
 		ginkgo.By("changing TCP service back to type=ClusterIP")
 		_, err = tcpJig.UpdateService(func(s *v1.Service) {
 			s.Spec.Type = v1.ServiceTypeClusterIP
-			s.Spec.Ports[0].NodePort = 0
```

Comment:
Any test we could add to show the new behavior?

Reply:
Great point, will add verification

Reply:
Yeah still thinking about the change, but I think a unit test would be better?

Reply:
We have unit tests and this makes e2e also verify this behavior.